src/module/classes/GmScreenApplication.ts

```diff
@@ -427,6 +427,12 @@ export class GmScreenApplication extends Application {
     this.updateCSSPropertyVariable(html, '.gm-screen-grid-cell', 'width', '--this-cell-width');
 
     const vanillaGridElement = document.querySelector('.gm-screen-grid');
+    // return early if suddenly no grids were found.
+    // this may happen in player view after a shared grid was removed.
+    if (!vanillaGridElement) {
+      $('.gm-screen-app').remove();
```

Comment:
This doesn't feel good.
Instead of rendering then deleting the Application, we should do something earlier to prevent the render from happening at all.

Reply:
We talked a bit a bout this. It is at the moment a bit complicated, as we need to "unrender" the application. For now this may stay like this but I'll add a big to-do.